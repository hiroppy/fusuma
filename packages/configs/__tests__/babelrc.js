const babelrc = require('../src/babelrc');

describe('babelrc', () => {
  it('should match settings', () => {
    expect(babelrc()).toMatchSnapshot();
  });
});
