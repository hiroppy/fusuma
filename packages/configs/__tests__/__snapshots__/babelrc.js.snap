// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`babelrc should match settings 1`] = `
Object {
  "compact": true,
  "plugins": Array [
    "@babel/plugin-syntax-dynamic-import",
    "@babel/plugin-proposal-class-properties",
    "@babel/plugin-proposal-object-rest-spread",
    Array [
      "prismjs",
      Object {
        "css": false,
        "languages": Array [],
        "plugins": Array [],
        "theme": "default",
      },
    ],
  ],
  "presets": Array [
    Array [
      "@babel/preset-env",
      Object {
        "modules": false,
        "targets": Object {
          "browsers": "last 1 version",
        },
      },
    ],
    "@babel/preset-react",
  ],
}
`;
